dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgProductCategory.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the LKSG questionnaire regarding a single product category for the "Product Categories" field
+ */
+data class LksgProductCategory(
+    @field:JsonProperty(required = true)
+    val definitionProductTypeService: List<String>,
+
+    val suppliersPerCountry: List<LksgCountryAssociatedSuppliers>?,
```

Comment:
`numberOf...` ?   What do you think?

Reply:
Absolutely. Will be regarded when switching to a map.

Reply:
Ok I will wait

Reply:
To be pushed

Reply:
👍